app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express")
+const app = express()
+app.use(express.json())
```

Comment:
Question: Is this middleware needed at this stage where we only have a `GET` request? When do we need this middleware (hint: it's not yet).

We should only introduce new middleware, packages etc as and when we need them.